src/main/java/io/bitrise/trace/step/InjectTraceTask.java

```diff
@@ -265,6 +277,24 @@ static boolean updateBuildScriptContent(final String path) throws IOException {
         return false;
     }
 
+    /**
+     * Finds and returns the positions of String literals in a given code content.
+     *
+     * @param codeContent the code content to examine.
+     * @return a List of {@link Range}s.
+     */
+    static List<Range> findStringLiterals(final String codeContent) {
```

Comment:
nit: it'd be nice to have a test for when this should find no results

Reply:
Added test case.